deepmd/common.py

```diff
@@ -483,3 +485,39 @@ def get_np_precision(precision: "_PRECISION") -> np.dtype:
         return np.float64
     else:
         raise RuntimeError(f"{precision} is not a valid precision")
+
+def load_model_info(model_file: str) -> Tuple[dict, float]:
```

Comment:
I would recommend split this function into two: one for loading json data with a given list of tensor names, and the other for loading any variable with a given list of tensor names. 

Both of them then can be reused in other cases where tensors with certain names are required to be evaluated.